aten/src/ATen/native/TensorShape.cpp

```diff
@@ -1311,6 +1311,28 @@ std::vector<Tensor> unsafe_split(const Tensor& self, int64_t split_size, int64_t
   return result;
 }
 
+std::vector<Tensor> hsplit(const Tensor& self, int64_t split_size, c10::optional<int64_t> sentinel) {
+  TORCH_CHECK(!sentinel.has_value(), "torch.hsplit takes either a single split value indicating the number of times the tensor is to be split or a list of indices where the input is to be split, but got two indices outside of a list!")
```

Comment:
"... the number of times the tensor ... " 

The use of the word "tensor" here should made consistent with the use of the word "input" later. That is, either both should be "input," both should be "tensor," or both should "input tensor."